CHANGELOG.md

```diff
@@ -183,6 +165,33 @@
 - `[internal/state]` Moved function `MedianTime` to package `types`,
   and made it a method of `Commit` so it can be used by external packages.
   ([\#2397](https://github.com/cometbft/cometbft/pull/2397))
+- `[privval]` allow privval to sign arbitrary bytes
+  ([\#2692](https://github.com/cometbft/cometbft/pull/2692))
+- Bump minimum Go version to v1.22
+  ([\#2725](https://github.com/cometbft/cometbft/pull/2725))
+- Remove `cleveldb` and `boltdb` ([\2786](https://github.com/cometbft/cometbft/pull/2786))
+- [`proto`] Remove `abci.ValidatorUpdate.pub_key`, add `pub_key_type` and
+  `pub_key_bytes` ([\#2843](https://github.com/cometbft/cometbft/pull/2843))
+- [`abci/types`] Replace `ValidatorUpdate.PubKey` with `PubKeyType` and
+  `PubKeyBytes` to allow applications to avoid implementing `PubKey` interface.
+  ([\#2843](https://github.com/cometbft/cometbft/pull/2843))
+ - [`abci/types`] Rename `UpdateValidator` to `NewValidatorUpdate`, remove
+   `Ed25519ValidatorUpdate` ([\#2843](https://github.com/cometbft/cometbft/pull/2843))
+- [`proto/privval`]  Replace `pub_key` with `pub_key_type` and `pub_key_bytes` in
+  `PubKeyResponse` ([\#2878](https://github.com/cometbft/cometbft/issues/2878))
+- [`proto/types`] Deprecate `pub_key` in favor of `pub_key_type` and `pub_key_bytes` in
+  `Validator` ([\#2878](https://github.com/cometbft/cometbft/issues/2878))
+- [`config`] Remove `skip_timeout_commit` in favor of `timeout_commit=0`
+  ([\#2892](https://github.com/cometbft/cometbft/pull/2892))
+- [`config`] Merge `timeout_prevote` and `timeout_precommit`,
+  `timeout_prevote_delta` and `timeout_precommit_delta` into `timeout_round`
+  and `timeout_round_delta` accordingly
+  ([\#2895](https://github.com/cometbft/cometbft/pull/2895))
+`[abci/client]` Deprecate `SetResponseCallback(cb Callback)` in the `Client` interface as it is no
+longer used. ([\#3084](https://github.com/cometbft/cometbft/issues/3084))
+`[mempool]` Change the signature of `CheckTx` in the `Mempool` interface to
+`CheckTx(tx types.Tx, sender p2p.ID) (*abcicli.ReqRes, error)`.
+([\#1010](https://github.com/cometbft/cometbft/issues/1010), [\#3084](https://github.com/cometbft/cometbft/issues/3084))
```

Comment:
The original file in `.changelog` dir has to be correctly formatted, and `unclog` called again.

Reply:
fixed